beluga_amcl/include/beluga_amcl/amcl_nodelet.hpp

```diff
@@ -55,6 +54,17 @@ class AmclNodelet : public nodelet::Nodelet {
  protected:
   void onInit() override;
 
+  auto get_initial_estimate() const -> std::optional<std::pair<Sophus::SE2d, Eigen::Matrix3d>>;
+
+  auto get_motion_model(std::string_view) const -> beluga_ros::Amcl::motion_model_variant;
+
+  auto get_sensor_model(std::string_view, const nav_msgs::OccupancyGrid::ConstPtr&) const
+      -> beluga_ros::Amcl::sensor_model_variant;
+
+  static auto get_execution_policy(std::string_view) -> beluga_ros::Amcl::execution_policy_variant;
```

Comment:
@nahueespinosa nit, arguably late: doesn't this belong to `beluga_ros`?

Reply:
I think it could, but I'm not sure what form it should take. Let me come back to this in the future. For now, I like the consistency with other variant filter arguments.